src/CGIHandler.cpp

```diff
@@ -176,49 +133,35 @@ std::string CGIHandler::executeCGI(const MetaVariables &env)
 		if (access(argvPointers[0], X_OK) == -1)
 		{
 			perror("access");
+			return false;
 			_exit(EXIT_FAILURE);
+			// TODO: @leo I don't think we should exit here. We don't want to kill the whole server cause of a CGI
```

Comment:
Why don't we set an appropriate error message and error code here? Something like `access denied`

Reply:
We can. I'm not the master of error codes. Tell me what is the correct error code. Are we checking here the interpreter or the requested script? @lmangall If you tell me which error should be set I can just change it.